packages/runner/lib/monitor.ts

```diff
@@ -79,17 +74,18 @@ export class RunnerMonitor {
             if (!environmentId || !activityLogId) {
                 continue;
             }
-            await httpFetch({
-                method: 'post',
-                url: `${this.persistServiceUrl}/environment/${environmentId}/log`,
-                headers: {
-                    Authorization: `Bearer ${secretKey}`
-                },
-                data: JSON.stringify({
+            await persistClient.postLog({
+                secretKey,
+                environmentId,
+                data: {
                     activityLogId: activityLogId,
-                    level: 'warn',
-                    msg: `Memory usage is high: ${memoryUsagePercentage.toFixed(2)}% of the total available memory.`
-                })
+                    log: {
+                        type: 'log',
+                        level: 'warn',
+                        message: `Memory usage is high: ${memoryUsagePercentage.toFixed(2)}% of the total available memory.`,
+                        createdAt: new Date().toISOString()
+                    }
```

Comment:
it was broken no?

Reply:
it was using the legacy log format which you just removed in https://github.com/NangoHQ/nango/commit/3dced1531cad838f13c7ddd14b3995290d392062#diff-871ef30263b732ac8701c666b7d42fcf2268407a048ae682633c42536a6a06fdL11-L19
so yes it was broken but not for a very long time